fe/fe-core/src/main/java/com/starrocks/connector/CatalogConnectorMetadata.java

```diff
@@ -128,6 +128,11 @@ public List<RemoteFileInfo> getRemoteFileInfos(Table table, List<PartitionKey> p
         return normal.getRemoteFileInfos(table, partitionKeys, snapshotId, predicate, fieldNames, limit);
     }
 
+    @Override
+    public boolean warmupMetadata(WarmupEntry warmupEntry) {
```

Comment:
WarmupEntry is not inappropriate, it will confuse with datacache warmup

Reply:
okay, maybe named it prepareMetadata